README.md

```diff
@@ -17,20 +17,20 @@ Dolphin Scheduler Official Website
 
 ### Design features:
 
-A distributed and easy-to-extend visual DAG workflow scheduling system. It dedicates to solving the complex dependencies in data processing to make the scheduling system `out of the box` for data processing.
+Dolphin Scheduler is a distributed and easy-to-extend visual DAG workflow scheduling system. It dedicates to solving the complex dependencies in data processing to make the scheduling system `out of the box` for the data processing process.
 
 Its main objectives are as follows:
 
- - Associate the tasks according to the dependencies of the tasks in a DAG graph, which can visualize the running state of task in real time.
- - Support many task types: Shell, MR, Spark, SQL (mysql, postgresql, hive, sparksql), Python, Sub_Process, Procedure, etc.
+ - Associate the tasks according to the dependencies of the tasks in a DAG graph, which can visualize the running state of the task in real-time.
+ - Support many task types: Shell, MR, Spark, SQL (MySQL, PostgreSQL, hive, spark SQL), Python, Sub_Process, Procedure, etc.
  - Support process scheduling, dependency scheduling, manual scheduling, manual pause/stop/recovery, support for failed retry/alarm, recovery from specified nodes, Kill task, etc.
- - Support process priority, task priority and task failover and task timeout alarm/failure
+ - Support process priority, task priority and task failover, and task timeout alarm/failure
  - Support process global parameters and node custom parameter settings
  - Support online upload/download of resource files, management, etc. Support online file creation and editing
  - Support task log online viewing and scrolling, online download log, etc.
  - Implement cluster HA, decentralize Master cluster and Worker cluster through Zookeeper
- - Support online viewing of `Master/Worker` cpu load, memory
- - Support process running history tree/gantt chart display, support task status statistics, process status statistics
+ - Support online viewing of `Master/Worker` CPU load, memory
+ - Support process running history tree/Gantt chart display, support task status statistics, process status statistics
```

Comment:
- Support graphing history tree/Gantt chart, task status statistics, and process status statistics (to the workflow)
might be better.

Reply:
Hi, 
"Support presenting tree or Gantt chart of workflow history as well as the statistics results of task & process status in each workflow." 
Could you please check if this version clearly and precisely shows the original meaning? I am afraid I misunderstood this sentence to the other side.